crates/runtime/src/task_history.rs

```diff
@@ -50,7 +50,7 @@ pub(crate) struct TaskSpan {
 
     pub(crate) task: Arc<str>,
     pub(crate) input: Arc<str>,
-    pub(crate) truncated_output: Option<Arc<str>>,
+    pub(crate) captured_output: Option<Arc<str>>,
```

Comment:
cc @ewgenius this will need a corresponding change in the cloud

Reply:
I’ll make sure this is handled